plugins/org.jboss.tools.livereload.core/META-INF/MANIFEST.MF

```diff
@@ -36,7 +36,10 @@ Require-Bundle: org.eclipse.ui,
  org.eclipse.jst.server.tomcat.core;bundle-version="1.1.500",
  com.fasterxml.jackson.core.jackson-annotations;bundle-version="2.5.0",
  com.fasterxml.jackson.core.jackson-core;bundle-version="2.5.0",
- com.fasterxml.jackson.core.jackson-databind;bundle-version="2.5.0"
+ com.fasterxml.jackson.core.jackson-databind;bundle-version="2.5.0",
+ org.apache.aries.spifly.dynamic.bundle;bundle-version="1.0.2",
+ org.apache.aries.util;bundle-version="1.1.1",
+ org.objectweb.asm.all;bundle-version="5.0.4"
```

Comment:
@alexeykazakov @xcoulon When I test sims I don't add aries to required bundles, so I don't think we need it here